### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][Eclipse][Wipro]:Unable to resize the columns in "Your Subscriptions" window for Azure signin with interactively.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1366]
**Repro Steps:**

1. Open and Start Eclipse.
2. Open your project with Eclipse.
3. Press "Alt+T" to open  Tools menu, Navigate to  "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and navigate to "Sign In" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. In "Your Subscriptions "dialog , tab to subscription and try to resize the column's

**Expected Result:**
Keyboard users should able to do resize by using keyboard

**Actual Result:**
Keyboard user's unable to do resize the columns by using keyboard in Your subscriptions dialog

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(Os Build 16299.192)

![unable to resize the columns in tour subscriptions window by using keyboard](https://user-images.githubusercontent.com/22956130/35679278-6f10839e-077c-11e8-9bff-c22609bd1728.jpg)

**Comment by Eskibear** [comment]
under investigation

**Comment by SummerSun** [comment]
Same as issue #1368

**Comment by Eskibear** [comment]
external issue: 
Ref:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=308107

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749824
